scalableminds/webknossos
# Assert unique tasktype name aka summary on update

This previously yielded a raw SQL error

### URL of deployed dev instance (used for testing):
- https://tasktypeupdatenamecheck.webknossos.xyz

### Steps to test:
- Create a second task type (with new name)
- Edit it, change the name to a new one, should be fine
- Edit it, change the name to a conflicting one, should yield readable error message
- Edit anything else, should be fine
